docs/pages/extensions/snapshot/index.md

```diff
@@ -141,6 +160,90 @@ application_url:
   required: false
 ```
 
+## Customizing Snapshot Workflow Triggers in MeshMap Snapshot
+You can configure your workflows to run when specific activity on GitHub happens, at a scheduled time, or when an event outside of GitHub occurs.
+
+### About events that trigger workflows
+GitHub Actions provides a variety of events that can trigger workflows, allowing you to automate your software development process. Each event corresponds to a specific activity, such as creating a pull request, pushing code to a repository, or releasing a new version.
+
+## Supported Events
+
+GitHub Actions supports the following events:
+
+- [`push`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push-event): Triggers when code is pushed to the repository.
+
+- [`pull_request`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request-event): Triggers when a pull request is opened or updated.
+
+- [`pull_request_target`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target-event): Triggers when a pull request is opened against the repository's default branch.
+
+- [`workflow_run`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run-event): Triggers when a workflow is executed.
+
+- [`check_run`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#check_run-event): Triggers when a check run is requested or completed.
+
+- [`check_suite`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#check_suite-event): Triggers when a check suite is requested or completed.
+
+- [`create`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#create-event): Triggers when a branch or tag is created.
+
+- [`delete`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#delete-event): Triggers when a branch or tag is deleted.
+
+- [`deployment`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#deployment-event): Triggers when a deployment is created or updated.
+
+- [`deployment_status`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#deployment_status-event): Triggers when a deployment's status is updated.
+
+- [`fork`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#fork-event): Triggers when a fork is created.
+
+- [`gollum`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#gollum-event): Triggers when a Wiki page is created or updated.
+
+- [`issue_comment`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#issue_comment-event): Triggers when an issue comment is created or edited.
```

Comment:
Repeating all of GitHub's workflow triggers here is not ideal in that as more are added or some deprecated, we will have to maintain and this page. Be sure to point out that triggers may come and go and that users should refer to GitHub documentation for more. Having this many links to GitHub docs is a little silly.

Reply:
Thanks for your feedback @leecalcote . I agree that it's not ideal to repeat all of GitHub's workflow triggers in the documentation for the MeshMap Snapshot Action. As you mentioned, the list of triggers is constantly changing, and it would be a lot of work to keep the documentation up-to-date.

I'll update the documentation to simply point users to the GitHub documentation for a complete list of triggers.